mmda/predictors/__init__.py

```diff
@@ -0,0 +1,18 @@
+# flake8: noqa
+from necessary import necessary
+
+from mmda.predictors.heuristic_predictors.dictionary_word_predictor import DictionaryWordPredictor
+
+__all__ = ['DictionaryWordPredictor']
+
+with necessary('pysbd', soft=True) as PYSBD_AVAILABLE:
+    if PYSBD_AVAILABLE:
+        from mmda.predictors.heuristic_predictors.sentence_boundary_predictor \
+            import PysbdSentenceBoundaryPredictor
+        __all__.append('PysbdSentenceBoundaryPredictor')
+
+with necessary(["layoutparser", "torch", "torchvision", "effdet"], soft=True) as PYTORCH_AVAILABLE:
```

Comment:
It would be nice to not have to define this in three places:

1. setup extras_require
2. required_backends on the predictor itself
3. here

But I don't have a good suggestion.

Reply:
yeah :/ ive created an issue here: https://github.com/allenai/mmda/issues/144

i need to think a bit more how this would be resolved. a reasonable stop-gap could be defining a top-level config that all 3 locations imports from; reserve for next PR